src/analog/commands/uplink/getChannelsTypes.ts

```diff
@@ -64,4 +77,7 @@ export const fromBytes = ( data: types.TBytes ): IChannelsMap => ({channelsMap:
  * @param parameters - command payload
  * @returns full message (header with body)
  */
-export const toBytes = ( {channelsMap}: IChannelsMap ): types.TBytes => (command.toBytes(id, channelsMap));
+export const toBytes = ( {channels}: IChannels ): types.TBytes => (command.toBytes(
```

Comment:
I would prefer this way:

```js
export const toBytes = ( {channels}: IChannels ): types.TBytes => (
    command.toBytes(
        id,
        channels.map(channel => channel.type)
    )
);
```

this is even better (as code is rather simple and to need to split to many lines):

```js
export const toBytes = ( {channels}: IChannels ): types.TBytes => (
    command.toBytes(id, channels.map(channel => channel.type))
);
```

Reply:
fixed